api/src/config.ts

```
@@ -7,6 +7,12 @@ import { SnakeNamingStrategy } from './utils/snake-naming-strategy';
 const env = process.env;
 const getBoolOrDefault = (value: string, defaultValue: boolean) => (value ? value === 'true' : defaultValue);
 
+const getNumberOrDefault = (value: string, defaultValue: number) => {
+  if (!value) return defaultValue;
+  const parsed = parseInt(value, 10);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)